xml/System.Security.Cryptography.Pkcs/SignedCms.xml

```diff
@@ -688,12 +660,13 @@
 -   <xref:System.Security.Permissions.KeyContainerPermissionFlags.Open?displayProperty=nameWithType>  
   
 -   <xref:System.Security.Permissions.KeyContainerPermissionFlags.Sign?displayProperty=nameWithType>  
-  
+
  ]]></format>
         </remarks>
-        <exception cref="T:System.ArgumentNullException">A null reference was passed to a method that does not accept it as a valid argument.</exception>
+        <exception cref="T:System.ArgumentNullException"><paramref name="signer" /> is <see langword="null" />.</exception>
         <exception cref="T:System.Security.Cryptography.CryptographicException">A cryptographic operation could not be completed.</exception>
-        <exception cref="T:System.InvalidOperationException">The value of the <paramref name="silent" /> parameter is <see langword="true" /> and a signing certificate is not specified.</exception>
+        <exception cref="T:System.InvalidOperationException">.NET Framework only: The value of the <paramref name="silent" /> parameter is <see langword="true" />, and a signing certificate is not specified.</exception>
```

Comment:
this is a bit confusing. If silent is not used, how come its value can be used to throw an exception?